Products/ZenUtils/zenpack.py

```diff
@@ -445,7 +445,7 @@ def restore(self):
         # Respect the source of the pack
         zpsToSort = {}
 
-        pattern = '(ZenPacks\.zenoss\.[a-zA-Z\.]*)'
+        pattern = '(ZenPacks\.[a-zA-Z]\.[a-zA-Z\.]*)'
```

Comment:
I think this will only match names that use a single character between the periods, e.g. `ZenPacks.a.name`.
Shouldn't this pattern be: `(ZenPacks\.[a-zA-Z]+\.[a-zA-Z\.]*)`?

Reply:
Yep.  You could actually remove the interior pattern and replace the second pattern * with + to simplify this more.